assets/src/edit-story/masks/index.js

```diff
@@ -165,6 +285,111 @@ export const MASKS = [
     path: CLIP_PATHS[MaskTypes.BLOB_8],
     ratio: 1,
   },
+  {
+    type: MaskTypes.BLOB_9,
+    name: __('Blob 9', 'web-stories'),
+    path: CLIP_PATHS[MaskTypes.BLOB_9],
+    iconPath: ICON_CLIP_PATHS[MaskTypes.BLOB_9],
+    ratio: 1,
```

Comment:
Based on the recent experience: it's important that a shape is fits correctly and "fully" into the `1:1` square or a `1:(1/ratio)` rectangle. Roughly it means that the shape touches each side of such a rectangle.